templates/_helpers.yaml

```diff
@@ -78,12 +78,16 @@
   {{ end }}
 {{- end }}
 
-{{ define "airflow_url" -}}
-{{ .Release.Name }}-airflow.{{ .Values.ingress.baseDomain }}
+{{ define "deployments_host" -}}
+  {{ printf "deployments.%s" .Values.ingress.baseDomain }}
 {{- end }}
 
-{{ define "flower_url" -}}
-{{ .Release.Name }}-flower.{{ .Values.ingress.baseDomain }}
+{{ define "old_airflow_url" -}}
```

Comment:
@andriisoldatenko what do you think about using `airflow_host`, `airflow_url`, `flower_host` and `flower_url` for these helper names? Instead of using `old` here?

Looks like we do a combination of creating urls/paths inside our helpers, as well as directly inline in `ingress.yaml`. We could re-use the helpers there, as well as in the `configmap.yaml`.

Reply:
@schnie @ashb looks like last point before approve?
not sure that i get Greg's point.

Reply:
@schnie after chatting with @ashb i think i understand your point:
refactor `<.Release.Name>/airflow` to `(include airflow-path .)`?
IMO ver1 `{{ .Release.Name>/airflow }}` is more readable than,
ver2: `{{ printf "%s/health" (include airflow-path .) }}`